spec/features/google_tag_manager_can_anonomise_page_url_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.feature 'Google Tag Manager' do
+  context 'When searching for jobs with home postcode filter' do
+    scenario 'parameters are removed from URL before sending URL to dataLayer', js: true do
+      expect(Geocoder).to receive(:coordinates).with('CH52DD')
+                                               .and_return([51.6622925, -0.1180655])
+
+      visit jobs_path
+
+      within '.filters-form' do
+        fill_in 'location', with: 'CH52DD'
+        select 'Within 5 miles'
+        page.find('.govuk-button[type=submit]').click
+      end
+
+      expect(evaluate_script('dataLayer')).to include(include('event' => 'parametersRemoved', 'dePIIedURL' => '/jobs'))
```

Comment:
Why are there two includes? (New to me)

Reply:
These are nested within the dataLayer which is an array as attributes of an object within the dataLayer.

The outer include says look at any object in the array.
The inner include says only care about these 2 attributes within each object.